flexmeasures/api/common/schemas/sensor_data.py

```diff
@@ -344,11 +345,23 @@ def load_bdf(sensor_data: dict) -> BeliefsDataFrame:
         source = get_or_create_source(current_user)
         num_values = len(sensor_data["values"])
         event_resolution = sensor_data["duration"] / num_values
-        dt_index = pd.date_range(
-            sensor_data["start"],
-            periods=num_values,
-            freq=event_resolution,
-        )
+
+        if event_resolution == timedelta(hours=0):
+            if num_values <= 1:
+                raise NotImplementedError(
+                    "Cannot save multiple instantaneous values simultaneously."
+                )
```

Comment:
The message suggests that this error is raised when the number of values is higher than 1, rather than lower or equal.